velox/functions/sparksql/tests/MapTest.cpp

```diff
@@ -105,5 +106,122 @@ TEST_F(MapTest, ErrorCases) {
       {inputVector1, inputVector2, inputVector11, inputVector22},
       true);
 }
+
+TEST_F(MapTest, complexTypes) {
+  auto rowKey = makeRowVector({
+      makeFlatVector<int64_t>(1, [](auto row) { return row; }),
+  });
+  auto rowValue = makeRowVector({
```

Comment:
I'm seeing that these test cases use same values for keys and maps. If there are bugs that mixup keys and values these tests won't catch them. Could you change to use different values for keys and maps.

Reply:
Get it, updated.